lib/Runtime/PlatformAgnostic/Platform/Windows/UnicodeText.cpp

```diff
@@ -408,6 +408,18 @@ namespace PlatformAgnostic
                     {
                         return true;
                     }
+#ifdef NTBUILD
```

Comment:
Why NTBUILD? This could also be gated on INTL_WINGLOB, which I think is a bit more specific, but then again this entire file (except for two functions) is gated on INTL_WINGLOB, so I think it can just be free-standing.

Reply:
I agree it's a bit odd, but the failfast in the caller is gated on NTBUILD so just following that same codepath. I'm not sure if INTL_WINGLOB or something else would be more or less relaxed. Removing the NTBUILD of course means that this failfast will trigger in ChakraCore builds as well. Another note: this entire file is INTL_WINGLOB though (as opposed to the `.ICU.cpp` PlatformAgnostic alternative).

Reply:
I should probably add a comment to the code explaining this, rather than relying on someone down the road to blame the lines and read the commit message.

Reply:
The entire file isn't WinGlob -- some of the functions there use NLS and are preferred even in ICU-enabled builds of Chakra/Core over the ICU equivalents. Its complicated.

Reply:
Also, I am of the opinion that failing fast early in ChakraCore is good, since if we don't failfast here, we will still failfast/assert later.

Reply:
Agreed.